croncheck.php

```diff
@@ -40,6 +40,7 @@
 }
 // @codingStandardsIgnoreEnd
 require('../../../config.php');
+require_once('icinga.php');
```

Comment:
Technically I think this should be called 'nagios' and not 'icinga'

Icinga leverages nagios compatible plugins, ie think of nagios more as the interface and icinga is the implementation (not quite true but close enough)

So I think change all references of icinga to nagios in the code. The comments could either swap or refer to both

It can't hurt to have a comment reference to the spec here:

https://assets.nagios.com/downloads/nagioscore/docs/nagioscore/3/en/pluginapi.html

Reply:
ha I just realized there already was some comments which are probably better

 *  - http://nagios.sourceforge.net/docs/3_0/pluginapi.html
 *  - https://nagios-plugins.org/doc/guidelines.html#PLUGOUTPUT